fontforgeexe/collabclientui.h

```diff
@@ -165,7 +165,11 @@ extern int Collab_getLastChangedPos( void );
 extern int Collab_getLastChangedCodePoint( void );
 
 
-typedef void (*collabclient_notification_cb)( FontViewBase* fv );
+/**
+ * See https://developer.gnome.org/gobject/stable/signal.html
+ * for implied parameters in GLib callback functions.
+ */
```

Comment:
Did this change, or were we just doing it wrong? And would this possibly be responsible for that mysterious crash that I can't quite remember?